### DISCUSSION TITLE
Copilot has stopped working in one specific project

### DISCUSSION THREAD

**Discussion by chrispytoes** [community/community#46514] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have a Next.js project in where Copilot was initially working fine, but at some point it just stopping giving any suggestions. I can open any other project and it works perfectly, so I'm definitely logged in. Just this one project has stopped working.

I'm using JetBrains Webstorm.

**Comment by Akash1134** [comment]
Sorry for the trouble here @chrispytoes , please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-a-jetbrains-ide
